bincache/src/strategies/hybrid.rs

```diff
@@ -233,6 +233,19 @@ impl CacheStrategy for Hybrid {
         }
         Ok(())
     }
+
+    fn get_cache_capacity(&self) -> Option<CacheCapacity> {
+        if let (Some(memory_byte_limit), Some(disk_byte_limit)) =
+            (self.memory_limits.byte_limit, self.disk_limits.byte_limit)
+        {
+            Some(CacheCapacity::new(
+                memory_byte_limit + disk_byte_limit,
+                self.memory_limits.current_byte_count + self.disk_limits.current_byte_count,
+            ))
+        } else {
+            None
+        }
+    }
```

Comment:
It has been decided by the sacred tradition of the fair coin toss, that we keep it this way.
I am very unhappy about it grrr but the coin has spoken.